### PR TITLE
LTD-5439-fix-dependabot-alerts

### PR DESCRIPTION
**Pull request by markj0hnst0n**
### Aim

Resolve dependencies to those recommended by dependabot [here](https://github.com/uktrade/lite-api/security/dependabot).

Deployed to dev2 environment.

[LTD-5439](https://uktrade.atlassian.net/browse/LTD-5439)

[LTD-5439]: https://uktrade.atlassian.net/browse/LTD-5439?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by Tllew — APPROVED** [review]
seems reasonable

File: Pipfile [hunk 1]
```diff
@@ -76,6 +76,7 @@ psycopg = "~=3.1.18"
 django-log-formatter-asim = "~=0.0.5"
 dbt-copilot-python = "~=0.2.1"
 dj-database-url = "~=2.2.0"
+certifi = "~=2024.7.4"
```
Comment (depsiatwal) [inline Pipfile:79]: any ideas where certifi came from ?

Comment (markj0hnst0n) [reply Pipfile:79]: I added it in as one of the other packages had it as a dependency.  If I add it here explicitly then the other package should use it.

**Review by hnryjmes — COMMENTED** [review]
has this branch been deployed to an environment to check the dependencies build properly?

**Comment by markj0hnst0n** [comment]
Not yet.  I'm still trying to work out why the e2e tests are failing @hnryjmes

**Comment by markj0hnst0n** [comment]
@hnryjmes I've deployed this to dev2 now and am making some checks.  Looks okay so far.

**Review by depsiatwal — APPROVED** [review]
lgtm

**Pull request merged**